Core/Date.cs

```diff
@@ -30,315 +30,138 @@ namespace System
 
-		public static bool IsLeapYear(int year)
-		{
-			return DateTime.IsLeapYear(year);
-		}
+		public static bool IsLeapYear(int year) => DateTime.IsLeapYear(year);
 
-		public static Date Parse(string s)
-		{
-			return new Date(DateTime.Parse(s));
-		}
+		public static Date Parse(string s) => new Date(DateTime.Parse(s));
 
-		public static Date Parse(string s, IFormatProvider provider)
-		{
-			return new Date(DateTime.Parse(s, provider));
-		}
+		public static Date Parse(string s, IFormatProvider provider) => new Date(DateTime.Parse(s, provider));
 
-		public static Date Parse(string s, IFormatProvider provider, DateTimeStyles style)
-		{
-			return new Date(DateTime.Parse(s, provider, style));
-		}
+		public static Date Parse(string s, IFormatProvider provider, DateTimeStyles style) => new Date(DateTime.Parse(s, provider, style));
 
-		public static Date ParseExact(string s, string format, IFormatProvider provider)
-		{
-			return new Date(DateTime.ParseExact(s, format, provider));
-		}
-
-		public static Date ParseExact(string s, string format, IFormatProvider provider, DateTimeStyles style)
-		{
-			return new Date(DateTime.ParseExact(s, format, provider, style));
-		}
+		public static Date ParseExact(string s, string format, IFormatProvider provider) => new Date(DateTime.ParseExact(s, format, provider));
 
-		public static Date ParseExact(string s, string[] formats, IFormatProvider provider, DateTimeStyles style)
-		{
-			return new Date(DateTime.ParseExact(s, formats, provider, style));
-		}
+		public static Date ParseExact(string s, string format, IFormatProvider provider, DateTimeStyles style) => new Date(DateTime.ParseExact(s, format, provider, style));
 
-		public TimeSpan Subtract(Date value)
-		{
-			return this - value;
-		}
+		public static Date ParseExact(string s, string[] formats, IFormatProvider provider, DateTimeStyles style) => new Date(DateTime.ParseExact(s, formats, provider, style));
 
-		public Date Subtract(TimeSpan value)
-		{
-			return this - value;
-		}
+		public TimeSpan Subtract(Date value) => this - value;
 
-		public string ToLongString()
-		{
-			return _dt.ToLongDateString();
-		}
+		public Date Subtract(TimeSpan value) => this - value;
 
-		public string ToShortString()
-		{
-			return _dt.ToShortDateString();
-		}
+		public string ToLongString() => _dt.ToLongDateString();
 
-		public override string ToString()
-		{
-			return ToShortString();
-		}
+		public string ToShortString() => _dt.ToShortDateString();
 
-		public string ToString(IFormatProvider provider)
-		{
-			return _dt.ToString(provider);
-		}
+		public override string ToString() => ToShortString();
 
-		public string ToString(string format)
-		{
-			if (format == "O" || format == "o" || format == "s")
-			{
-				return ToString("yyyy-MM-dd");
-			}
+		public string ToString(IFormatProvider provider) => _dt.ToString(provider);
 
-			return _dt.ToString(format);
-		}
+		public string ToString(string format) => _dt.ToString(format);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)